doc/reference/misc.md

```diff
@@ -1,4 +1,9 @@
-# 🦉 Miscellaneous 🦉
+# 🦉 Miscellaneous 🦉 
+
+## Content
+
+- [AsyncRoot](#asyncroot)
```

Comment:
I would put portal first actually

Reply:
also, do not forget to add the `AsyncRoot` term to the owl content list in `doc/readme.md`